### PR TITLE
Update main.json

### PR DESCRIPTION
**Pull request by luveluen**
This is the
main.json update for the

/options/inventory/index.jade
/options/inventory/inventory.jade
/options/inventory/stable.jade
update

### CODE REVIEW

File: locales/en/main.json [hunk 1]
```diff
@@ -617,6 +617,59 @@
...
+"noScrolls" : "You don't have any quest scrolls.",
+"scrollsText1" : "Quests require parties. If you want to quest solo,",
+"scrollsText2" : "create an empty party",
+"scrollsPre" : "You need to complete the previous quest to start this one!",
+"food" : "Food",
+"noFood" : "You don't have any food.",
+"special" : "Special",
+"alexander" : "Alexander the Merchant",
+"welcomeMarket" : "Welcome to the Market! Buy hard-to-find eggs and potions! Sell your extras! Commission useful services! Come see what we have to offer.",
+"sellEgg" : "Sell {{selectedEgg.key}} for {{selectedEgg.value}} Gold",
```
Comment (paglias) [inline locales/en/main.json]: mmm... @lefnire what do you think about hardcoding angular variables in translation strings like that? 

@luveluen I think we should make sure, at least for now, not to include those in the json files, better to split it into more strings:

```
=env.t('sell')
| {{selectedEgg.key}}
....
```

**Comment by luveluen** [comment]
I updated all files to not include variables within strings, following your example.
I also noticed 2 obsolete lines that I never used, so I got rid of them.

**Comment by paglias** [comment]
@luveluen the other pull request on habitrpg (https://github.com/HabitRPG/habitrpg/pull/2495) is updated?

**Comment by paglias** [comment]
Is update but has some conflicts, I'm going to try fixing it. In the meantime I will merge this one.

**Pull request merged**